harvest.yml

```diff
@@ -70,4 +70,13 @@ Pollers:
   #   ssl_key: /path/to/key.key
   #   ca_cert: /path/to/server.crt
   #   exporters:
-  #     - prometheus1
\ No newline at end of file
+  #     - prometheus1
+
+  # StorageGRID example
+  #  grid-01:
+  #    datacenter: DC-01
+  #    addr: 10.0.0.3
+  #    username: myuser
+  #    password: mypass
+  #    collectors:
+  #      - StorageGrid
```

Comment:
do we want to mention example of another prometheus which has storagegrid metrics exported.